client-gen/src/snapshots/sails_clientgen__generator__tests__rmrk_works.snap

```diff
@@ -9,15 +9,15 @@ use parity_scale_codec::{Decode, Encode};
 use sails_rtl::collections::BTreeMap;
 #[allow(unused_imports)]
 use sails_rtl::{
-    calls::{Remoting, RemotingAction},
+    calls::{Activation, Call, Remoting, RemotingAction},
     prelude::*,
     String,
 };
 pub mod ctor {
     use super::*;
     pub trait RmrkCatalogConstructors<A: Default> {
```

Comment:
I think it has to be under the `traits` module with a name like `RmrkCatalogFactory`, i.e. it should be pretty much the same as for service client